Gemfile

```diff
@@ -71,5 +71,8 @@ group :test do
   gem 'webdrivers'
 end
 
-gem 'rspec-rails', '~> 6.0', '>= 6.0.1'
 gem 'rubocop', '>= 1.0', '< 2.0'
+group :development, :test do
+  gem 'rspec-rails', '~> 6.0', '>= 6.0.1'
+end
```

Comment:
[OPTIONAL]
- You already have a group :development, test on line 51. I would suggest deleting lines 75-77 and placing the rspec-rails gem in the existing block because there is currently repetition.